src/rpc/util.cpp

```diff
@@ -419,8 +422,12 @@ RPCHelpMan::RPCHelpMan(std::string name, std::string description, std::vector<RP
 {
     std::set<std::string> named_args;
     for (const auto& arg : m_args) {
+        std::vector<std::string> names;
+        boost::split(names, arg.m_names, boost::is_any_of("|"));
         // Should have unique named arguments
-        CHECK_NONFATAL(named_args.insert(arg.m_name).second);
+        for (const std::string& name : names) {
```

Comment:
Since you create accessors for `m_names` (`GitName`, `GetFirstName`), it would make sense to make `m_names` private, no?

Plus this would allow you to have a function `GetNames` similar to:
```
std::vector<std::string> RPCArg::GetNames() const
{
    std::vector<std::string> names;
    boost::split(names, m_names, boost::is_any_of("|"));
    return names;
}
```
and do here more cleanly `for (const std::string& name : arg.GetNames()) {`

(note that `Fallback`, `m_fallback`, `m_description`, `m_oneline_description` can also be made private without any additional code change)

Reply:
`RPCArg` is a `struct`, because it was meant to be a stupid class to only hold data and not have complicated member functions. I see that with the name aliases this is no longer true. I think it can be made a class and the `GetNames` utility can be added. Though, I will leave this pull as is for now, because currently `GetNames` is only used in this one place. And just in-lining it will yield the least amount of LOC written.